docs/content/manual/pre-release/node-not-ready/kubelet-restart/kubelet-restart-on-a-node.md

```diff
@@ -5,54 +5,30 @@ title: Test kubelet restart on a node of the cluster
 
-### Case 1: Kubelet restart on RKE1 multi node cluster:
-1. Create a RKE1 cluster with config of 1 etcd/control plane and 3 worker nodes.
-2. Deploy Longhorn on the cluster.
-3. Deploy prometheus monitoring app on the cluster which is using Longhorn storage class or deploy a statefulSet with Longhorn volume.
-4. Write some data into the mount point and compute the md5sum.
-5. Restart the kubelet on the node where the statefulSet or Prometheus pod is running using the command `sudo docker restart kubelet`
-6. Observe the volume. It becomes degraded but is still running. 
-7. Once the node is back, the volume of the workload should work fine and the data is intact.
-8. Scale down then re-scale up the workload. Verify the existing data is correct.
-
-### Case 2: Kubelet restart on one node RKE1 cluster:
-1. Create a RKE1 cluster with config of 1 node all role.
-2. Deploy Longhorn on the cluster.
-3. Deploy prometheus monitoring app on the cluster which is using Longhorn storage class or deploy a statefulSet with Longhorn volume.
-4. Write some data into the mount point and compute the md5sum.
-5. Restart the kubelet on the node using the command `sudo docker restart kubelet`
-6. Check the instance manager pods on the node are still running.
-7. Observe the volume. It gets detached and the pod gets terminated (since the only replica of the volume becomes failed).
-8. Once the pod is terminated, a new pod should be created and get attached to the volume successfully.
-9. Verify that the mount of the volume is successful and data is safe.
+### Note:
+- This is a refine version for match current behavior
+- Using below command to restart kubelet
+   - systemctl restart k3s-agent
+   - systemctl restart rke2-agent
 
-### Case 3: rke2-server/rke2-agent restart on RKE2 multi node cluster:
-1. Create a RKE2 cluster with config of 1 control plane and 3 worker nodes.
+### Case 1: Restart Volume Node Kubelet Immediately
+1. Create cluster with config of 1 etcd/control plane and 3 worker nodes.
 2. Deploy Longhorn on the cluster.
-3. Deploy prometheus monitoring app on the cluster which is using Longhorn storage class or deploy a statefulSet with Longhorn volume.
+3. Deploy a statefulSet with Longhorn volume.
 4. Write some data into the mount point and compute the md5sum.
-5. Restart the rke-agent service on the node where the statefulSet or Prometheus pod is running using the command `systemctl restart rke2-agent.service`
-6. Observe the volume. It becomes degraded but is still running.
-7. Once the node is back, the volume of the workload should work fine and the data is intact.
-8. Scale down then re-scale up the workload. Verify the existing data is correct.
-9. Create a StatefulSet with Longhorn volume on the control plane node.
-10. Once the StatefulSet is up and running, Write some data into the mount point and compute the md5sum.
-11. Restart the rke2-service on the control plane node using the command `systemctl restart rke2-server.service`.
```

Comment:
The original manual test cases also include restarting the kubelet on the `control plane`. Should we implement them in robot as well?

Reply:
According to the Kubernetes doc, control plane nodes are tainted by default to prevent scheduling of regular workloads ([Control Plane Node Isolation](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#control-plane-node-isolation)). As it is our e2e test framework design too, I perfer not adding test on the control plane node. 
If it is necessary,  I’ve created a [ticket](https://github.com/longhorn/longhorn/issues/11095) to track this task, thank you. cc @longhorn/qa